### PR TITLE
Adapt to new gocql.ParseConsistency API

### PR DESCRIPTION
**Pull request by aliaksandrb**
Hi!

Should help with https://github.com/db-journey/cassandra-driver/issues/1

### CODE REVIEW

**Comment by aliaksandrb** [comment]
Hmm, tests are passing if I update working dir in docker-compose file to my repo, but by default it tries to run then against original. Is it correct? Any advice how to provide that PR correctly? :)

```bash
make test
docker-compose run --rm go-test
Starting cassandradriver_cassandra_1 ... done
github.com/db-journey/migrate (download)
github.com/alecthomas/template (download)
github.com/db-journey/migrate/direction
github.com/alecthomas/template/parse
github.com/alecthomas/template
github.com/db-journey/migrate/file
github.com/db-journey/migrate/driver
github.com/aliaksandrb/cassandra-driver
=== RUN   TestMigrate
--- PASS: TestMigrate (5.78s)
PASS
ok  	github.com/aliaksandrb/cassandra-driver	5.787s
```

**Comment by n0rmanc** [comment]
Any update?

File: cassandra.go [hunk 1]
```diff
@@ -176,19 +176,12 @@ func init() {
...
-			err, ok = r.(error)
-			if !ok {
-				err = fmt.Errorf("Failed to parse consistency \"%s\": %v", consistencyStr, r)
-			}
-		}
-	}()
-	consistency = gocql.ParseConsistency(consistencyStr)
+	consistency, err = gocql.ParseConsistency(consistencyStr)
+	if err != nil {
+		err = fmt.Errorf("Failed to parse consistency \"%s\": %v", consistencyStr, err)
```
Comment (gravis) [inline cassandra.go]: Please use `%q` instead of `\"%s\"`

**Comment by gravis** [comment]
Sorry for the delay, we're currently at GitHub Universe conference.
I'll merge this for now, and check out how we can fix the merge requests w/ docker-compose later.
Thanks!

File: cassandra.go (hunk 1 above)
Comment (gravis) [reply cassandra.go]: Could you just fix this before I merge please?

Comment (aliaksandrb) [reply cassandra.go]: Done

**Comment by gravis** [comment]
Sorry, I haven't seen #3 when I commented on this. #3 is even simpler, I merged it instead. Thanks for your work, and sorry for the mess.

**Pull request closed without merging**